mesonbuild/modules/python.py

```diff
@@ -491,11 +492,15 @@ def _check_version(self, name_or_path, version):
         return True
 
     @FeatureNewKwargs('python.find_installation', '0.49.0', ['disabler'])
+    @FeatureNewKwargs('python.find_installation', '0.51.0', ['modules'])
     @disablerIfNotFound
-    @permittedKwargs(['required'])
```

Comment:
orthogonal, but if we allow `native` here and pass it on to `ExternalProgram` I think it will work?

Reply:
I guess? I don't know why you'd want to find a python executable for !build though?

Reply:
@dcbaker then `ExternalProgram` should be given `MachineChoice.build` unconditionally as it defaults to the host platform like everything else I think.